release-2020.03.30 to develop

AndreMiras opened the pull request:
Resolved conflict locally, see log below (from `develop` branch):
```
$ git merge --no-ff release-2020.03.30 
Auto-merging tests/test_recipe.py
Auto-merging tests/test_bootstrap.py
CONFLICT (modify/delete): testapps/setup_testapp_python3.py deleted in HEAD and modified in release-2020.03.30. Version release-2020.03.30 of testapps/setup_testapp_python3.py left in tree.
Auto-merging pythonforandroid/recommendations.py
Auto-merging pythonforandroid/recipes/python3/__init__.py
Auto-merging pythonforandroid/recipes/protobuf_cpp/__init__.py
Auto-merging pythonforandroid/recipes/numpy/__init__.py
CONFLICT (modify/delete): pythonforandroid/python.py deleted in HEAD and modified in release-2020.03.30. Version release-2020.03.30 of pythonforandroid/python.py left in tree.
Auto-merging pythonforandroid/archs.py
Auto-merging .travis.yml
CONFLICT (content): Merge conflict in .travis.yml
Automatic merge failed; fix conflicts and then commit the result.
```

inclement commented:
I didn't see what conflicts you needed to resolve, but I'm happy to assume it wasn't something worth worrying about. Will leave you to confirm and merge though. Thanks!

AndreMiras commented:
Yes the conflict merge doesn't appear really and I've edited the actual merge commit to include the conflicting files and action taken. And also I realised `testapps/setup_testapp_python3.py` should have been deleted not modified.
What I suspect happened for `testapps/setup_testapp_python3.py` is we touched this file in the release branch via https://github.com/kivy/python-for-android/commit/24037dc9761e5102ad4c9a216a4b8ff759a6357c while it got deleted in develop while we were releasing, so conflict makes sense.
Then for `pythonforandroid/python.py` and `.travis.yml` I'm not sure, maybe when we merged master back to the release theses file were also altered somehow.
Merge commit message now contains:
```
git mergetool

This message is displayed because 'merge.tool' is not configured.
See 'git mergetool --tool-help' or 'git help config' for more details.
'git mergetool' will now attempt to use one of the following tools:
opendiff kdiff3 tkdiff xxdiff meld tortoisemerge gvimdiff diffuse diffmerge ecmerge p4merge araxis bc codecompare smerge emerge vimdiff
Merging:
.travis.yml
pythonforandroid/python.py
testapps/setup_testapp_python3.py

Normal merge conflict for '.travis.yml':
  {local}: modified file
  {remote}: modified file
Hit return to start merge resolution tool (vimdiff):
4 files to edit

Deleted merge conflict for 'pythonforandroid/python.py':
  {local}: deleted
  {remote}: modified file
Use (m)odified or (d)eleted file, or (a)bort? d

Deleted merge conflict for 'testapps/setup_testapp_python3.py':
  {local}: deleted
  {remote}: modified file
Use (m)odified or (d)eleted file, or (a)bort? d
```
I think we're good to go, could you give it another look?

inclement commented:
Nice :)
